vulnerable_people_form/integrations/location_eligibility.py

```diff
@@ -10,6 +10,16 @@
 _DEFAULT_TIER = 1
 
 
+def get_uprn_shielding(uprn, app):
```

Comment:
maybe rename these to get_shielding_advise_by_uprn and get_shielding_advise_by_postcode?